templates/managed/apps/index.html

```diff
@@ -85,18 +85,18 @@ <h1>Performant open source with Managed Apps</h1>
         </div>
         <div class="p-matrix__content u-align--center">
           <div class="p-matrix__desc">
-            <p class="p-muted-heading">EXTENSIBLE, COMPLIANT SQL</p>
+            <p class="p-muted-heading">Extensible, compliant SQL</p>
           </div>
         </div>
       </li>
       <li class="p-matrix__item u-align--center">
         <div style="min-width: 50%">
         {{
           image(
-          url="https://assets.ubuntu.com/v1/d12bc087-apps_promethus.png",
-          alt="promethus",
-          width="144",
-          height="90",
+          url="https://assets.ubuntu.com/v1/11e66fc8-Opensearch_Logo.svg",
+          alt="OpenSearch",
+          width="125",
+          height="25",
           hi_def=True,
           loading="lazy",
           attrs={"style": "height: 80px; width: auto;"},
```

Comment:
The image is rendering in a way that makes it look low res, this fixes it:
```suggestion
          alt="OpenSearch",
          width="200",
          height="40",
          hi_def=True,
          loading="lazy",
          attrs={"style": "margin-top:.8rem;"},
```